students/ssankura/lesson02/music_tracks.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+'''
+Lesson02 - Assignment 01 - Generators and Closures 
+Author: Sireesha Samkuratripati
+TASK1: Reads Music from Pandas, Filters Music based on criteria - Artist = 'Ed Shreeran' and prints the results
+TASK2: Implemented generator - filter_music_generator which returns one music record at a time with the data - Track name, artist name, energy level, danceability and loudness
+TASK3: Implemented Closure - music_high_energy - which takes as input the filename of the music input file feed - it had a nested func which filters music (from input file) based on energy level (sent as input)
+'''
+
+
+import pandas as pd
+dash = "-" *120
+
+def filter_music_pandas_iterrows():
+	''' Used comprehension to filter and print all the tracks by Ed Sheeran
+	'''
+	music = pd.read_csv("featuresdf.csv")
+	music.head()
+	music.describe()
```

Comment:
Your repeating the 3 lines above in all your functions.  Make it a function itself and call it.